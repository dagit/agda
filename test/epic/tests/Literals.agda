{-# OPTIONS --universe-polymorphism #-}
module tests.Literals where

open import Prelude.Nat
open import Prelude.Float
open import Prelude.Char
open import Prelude.String
open import Prelude.Unit
open import Prelude.IO

afloat : Float
afloat = 1.23

astring : String
astring = "abc"

achar : Char
achar = 'd'

anat : Nat
anat = 123

main : IO Unit
main =
  printFloat  afloat  ,,
  putStr      astring ,,
  printChar   achar   ,,
  printNat    anat    ,,
  putStrLn ""
