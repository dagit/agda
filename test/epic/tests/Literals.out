1.23abcd123
